README.md

```diff
@@ -32,8 +32,8 @@ git init
 - Replace references:
 
 ```bash
-grep -rl "r.pkg.template" --exclude-dir='.git' | xargs perl -p -i -e "s/r.pkg.template/${package_name}/g"
-grep -rl "insightsengineering" --exclude-dir='.git' | xargs perl -p -i -e "s/insightsengineering/${package_owner}/g"
+grep -rl "r.pkg.template" --exclude-dir='.git' * | xargs perl -p -i -e "s/r.pkg.template/${package_name}/g"
+grep -rl "insightsengineering" --exclude-dir='.git' * | xargs perl -p -i -e "s/insightsengineering/${package_owner}/g"
```

Comment:
Technically not needed as `grep -r` does a recursive search

Reply:
Hey @dinakar29 , thanks, I am getting `grep: warning: recursive search of stdin` if "*" is not used. I think either "." or "*" is needed here.

Reply:
Cool, that should be fine. We can leave this as-is (i.e. with `*`)